### PR TITLE
Review for Portuguese translation of `maintenance_cheatsheet.pt.Rmd`

### PR DESCRIPTION
**Pull request by pedropark99**
This PR brings three change to the automatic output generated by the translation tools:

- Fix small typos in some sentences;
- Rephrase a sentence in the `Promoção de problemas nos pacotes` section. The way the phrase is strucutured works in english, but did not worked in Portuguese, so that is why I rephrase it.
- Changed the use of the word "comiseração" (which was translated from "commiseration") to "papo". Because "comiseração" is an obscure word in our language, is not really used or known by many people. So I opted to use a much popular word that kinds of translates the idea (I'm open to better suggestions than "papo").

In respect to the word "cheatsheet", the API translated it into "Folha de dicas", which is not a bad choice, but maybe... just maybe, is better to use "Cola" instead, which is a more direct translation of the "cheatsheet" word. But I will leave this choice to whoever will do the Review 2.

### CODE REVIEW

**Comment by pedropark99** [comment]
@joaojcorreia and @danielvartan
This PR is ready for Review 2.

**Comment by danielvartan** [comment]
Great! I will have some time tomorrow to start the review.

**Comment by joaojcorreia** [comment]
Given that @danielvartan had already volunteered, I assume I won't be needed now. However, I suggest you keep "folha de dicas", as "cola" won't be understood in PT-PT, and i presume "cábula" won't be in PT-BR.

**Comment by pedropark99** [comment]
Good point @joaojcorreia , thank you!

**Comment by danielvartan** [comment]
> Changed the use of the word "comiseração" (which was translated from "commiseration") to "papo". Because "comiseração" is an obscure word in our language, is not really used or known by many people. So I opted to use a much popular word that kinds of translates the idea (I'm open to better suggestions than "papo"). @pedropark99 

Just leaving this here if someone is curious about it. 😂

---

### comiseração

__co·mi·se·ra·ção__
_sf_

__1__ Ato de comiserar-se.
__2__ Compaixão pelos infortúnios alheios.

Source: Michaelis \<https://michaelis.uol.com.br/busca?id=lW1m \>.

---

### comiserar

__co·mi·se·rar__
_vtd_
__1__ Fazer sentir comiseração ou piedade.
_vpr_
__2__ Sentir-se compadecido

Source: Michaelis \<https://michaelis.uol.com.br/busca?r=0&f=0&t=0&palavra=comiserar \>.

File: index.Rmd [hunk 1]
```diff
@@ -8,10 +8,11 @@
 #| echo: false
 #| results: 'asis' 
 #| warning: false
+#| eval: fase
```
Comment (danielvartan) [inline index.Rmd:11]: This should be `#| eval: false`.

**Comment by danielvartan** [comment]
@pedropark99 I just finished reading the [translation guidelines](https://translationguide.ropensci.org/). I plan to do the review tomorrow.

It's not clear to me what the preferred method for submitting changes in the 2nd review is. Do you prefer that I simply comment on the changes using the review tools in the PR, or commit the changes in the fork? In the latter case, according to the guidelines, I would need to have permission to write on it.

Regarding the title  (@joaojcorreia),  perhaps it's a good idea to use a solution closer to the Spanish translation used in rOpenSci Glossary (see [here](https://github.com/ropensci-review-tools/glossary/blob/120ff276f9a278c1f400172f11049449d8a1b372/glossary.csv#L49)), don't you think?

```
cheatsheet, guía rápida,,
cheat sheet, guía rápida,,
```

**Comment by joaojcorreia** [comment]
I would go with "guia rápido". As it is self-describing and I belive understandable for all.

**Comment by joaojcorreia** [comment]
Adding to my previous comment and addressing @danielvartan comment about "comiseração". I agree with Daniel, this is an obscure term not regularly used in day-to-day contexts. However, "papo" isn't in my view an adequate alternative. 

Just for context, "papo" is an ornithological term meaning a bird crop. In PT-BR the term evolved into meaning an open conversation, yet in PT-PT it still means a bird crop.  I would suggest "uma conversa amigável".

Additionally, I would avoid the usage of "você". "Você" is a contraction of "vossa mercê", an archaic form of respect. In PT-BR it evolved into a coloquial form of referring to "you", yet in PT-PT it evolved into a condescending way of exhibiting fake respect. I would suggest we abstain from using "você", and just conjugate the verb in the third person of the singular leaving the noun implicit.

Please don't misinterpret my comments. I understand that we may not always be fully aware of our individual use of localisms. If you wish, I would be happy to go over the document to tweak it to a neutral version of Portuguese.

**Comment by pedropark99** [comment]
Fist point, I liked the option of using "guia rápido".

Now, regarding the doubt of @joaojcorreia about the review process. The guidelines for review 2 talks about pulling the latest changes, and working in the branch associated with the file that is being translated. So I understand that you need to clone my fork, git switch to my branch that is associated with this PR, and do the changes there, and then, commit them. However, if you find appropriate, I do not have problems with you reviewing by just using the PR Review tools, which are quite handy. 

@yabellini can you help me to advise @joaojcorreia better in this subject?

**Comment by pedropark99** [comment]
Second point, I also liked the suggestion @joaojcorreia of using "conversa amigável" instead of "papo".

About the points raised about the word "você", when you say:
> and just conjugate the verb in the third person of the singular leaving the noun implicit.

I think it would add to the discussion if you give an example of a phrase in both states, using the third person, and not using the third person. This way you present your idea in the most clear way as possible.

Besides that, I understand that this idea of avoiding the use of "você" with the objective of reaching to a more "neutral" portuguese is aligned with the translation objectives. And for that, I think we will need great help of you @joaojcorreia  to reach this objective.

**Comment by joaojcorreia** [comment]
To materialize what I am saying about "você":

--------------------------------------------------------------------------------------
Você também pode usar o nosso canal de manutenção de pacotes no Slack.

Would become:

Pode, também, usar o nosso canal de manutenção de pacotes no Slack.
-----------------------------------------------------------------------------------

Você deve ter acesso administrativo ao repositório do GitHub do seu pacote.

would become:

Deverá ter acesso administrativo ao repositório do GitHub do pacote.

---------------------------------------------------------------------------------------

Se você tiver alguma pergunta ou solicitação sobre o GitHub (por exemplo, adicionar um colaborador à organização do GitHub), você pode usar um dos canais públicos no Slack do rOpenSci ou enviar uma mensagem para `@ropensci/admin` no GitHub.

would become:

Se tiver pergunta ou solicitação sobre o GitHub (por exemplo, adicionar um colaborador à organização do GitHub), deverá usar um dos canais públicos no Slack do rOpenSci ou enviar uma mensagem para `@ropensci/admin` no GitHub.

----------------------------------------------

I think this way it is clear for all.

**Comment by danielvartan** [comment]
Thank you for the responses, @pedropark99 and @joaojcorreia.

@pedropark99, regarding the changes, as you said, I will fork and commit them.

About the title. 'Guia rápido' has a verbal agreement issue, as it's a feminine noun. It should be 'guia rápida.'

Regarding 'você,' I respectfully disagree with you, @joaojcorreia. I understand your point, but we must also consider that the vast majority of Portuguese speakers are Brazilians. By omitting 'você' from the translations, te text looks very weird, approaching it to Portuguese speakers but distancing itself from Brazilian speakers.

As the guidelines also state, we must choose one style for the language, aiming to include all people in the process. However, we can't do this by undermining the primary style. In Spanish, this is the Latin American style. In Portuguese, this should be the Brazilian style.

**Comment by pedropark99** [comment]
@danielvartan raised a good point about "você". I understand we have two choices here in this subject:

1. Find a middle ground. That is, to find a word (or a sentence structure) that can substitute "você", and that is: 1) familiar to most brazilians; 2) also used in PT-PT; 3) a good-enough representation of "you".
2. Prioritize whatever choice that fits best into the translation guidelines and objetives.

Maybe @yabellini  can help us to decide which of these two options above we should pursue. I think that we should focus on the option 2.

**Comment by danielvartan** [comment]
I believe that @joaojcorreia is correct in arguing against the use of "você" in certain cases. My disagreement lies in the idea of completely avoiding it. There are sentences where a pronoun is unnecessary.

For example: [`Você precisa de ajuda?`](https://github.com/danielvartan/dev_guide/blob/f4fdff84d950b90f5e8bc69df389f06a10177900/maintenance_cheatsheet.pt.Rmd#L8) conveys the same meaning (at least for Brazilian speakers) as `Precisa de ajuda?`.

However, consider the case raised by @joaojcorreia:

[`Você deve ter acesso administrativo ao repositório do GitHub do seu pacote.`](https://github.com/danielvartan/dev_guide/blob/f4fdff84d950b90f5e8bc69df389f06a10177900/maintenance_cheatsheet.pt.Rmd#L17.) → `Deverá ter acesso administrativo ao repositório do GitHub do pacote.`. This simply doesn't sound natural.

In short, I believe that fluency should be one of the primary considerations, in line with the first general guideline (["Use a conversational voice rather than a formal or academic voice"](https://translationguide.ropensci.org/specific_guidelines.html#general-guidelines)). If you can maintain fluency without using pronouns, great! Don't use them. However, if a pronoun enhances fluency, then it should be used.

**Comment by danielvartan** [comment]
Jumping to another subject.

Are we using a masculine article or a feminine article when referring to rOpenSci?

I usually think rOpenSci as an organization. Since "organização" is a feminine noun, I believe it must be referred with the article "a".

For example: 

[Veja Documentos do rOpenSci](https://github.com/danielvartan/dev_guide/blob/f4fdff84d950b90f5e8bc69df389f06a10177900/maintenance_cheatsheet.pt.Rmd#L27).

Should be : 

Veja os documentos da rOpenSci.

**Comment by danielvartan** [comment]
@pedropark99 I have submitted a PR with the review changes. Please take a look at it when you have some time.

There were a lot of small things to consider. The next translations may be faster. 😁

**Comment by pedropark99** [comment]
Thank you for the review @danielvartan ! @maelle  I understand this PR is ready to merge (reviews 1 and 2 done) 😉.

**Comment by marianambc** [comment]
Hi, here Mariana (from Lisbon), happy to help with the review process as well. Thank you.

**Comment by yabellini** [comment]
Hello, thank you very much for all this work and the beautiful discussion about the best way to translate the guide. I'm sorry for not getting back to you sooner, lots of traveling this month. 

Each language is free to discuss and decide the rules for better localization and translation of the content.  You have already seen some of the rules we have in Spanish that can be used in the Portuguese translation.  For Spanish, we use some of the agreements that other translations did about Spanish.  Perhaps @beatrizmilz can share some guidelines with us? 

You already see the general rules (like conversational language, trying to avoid gender marks, etc), so I think these are very valuable discussions.  You are native speakers, so you are better equipped to make a decision than I am.

If you share with me your emails I will send an invite to our Slack (if you are not already there) and I will also create a channel for this translation, and you can discuss these details there too. 

We should write these rules and agreements in Portuguese and incorporate them into the guidelines.  

Thanks so much for doing this important work!

**Comment by beatrizmilz** [comment]
Hi!
This presentation lists other two translation projects:
https://docs.google.com/presentation/d/1uqsPo715R_r0u6wPyC3EBbUVkri0qjf7SV5Y-zTfTZ0/edit?usp=sharing


This is the guide for the R base translation:

https://contributor.r-project.org/translations/Conventions_for_Languages/Brazilian%E2%80%90Portugese-specific-translations.html

This is the guide for the translation of the R for Data Science book: https://github.com/cienciadedatos/pt-r4ds/wiki/3-%E2%80%90-Guia-de-contribui%C3%A7%C3%A3o-com-a-tradu%C3%A7%C3%A3o-do-livro-%22R-for-Data-Science-(2e)%22 (this is growing as we translate)

**Comment by danielvartan** [comment]
Great initiative, @beatrizmilz! I'll try to help as I can.

**Comment by marianambc** [comment]
Thank you for all the documents, I'll be looking into it later today and
tomorrow. Thanks!
